spec/views/groups/show.html.haml_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe "groups/show", type: :view do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.